### PR TITLE
[9.0] mgmtsystem

### PR DESCRIPTION
**Pull request by naousse**
changement

Migration du module

Création des dossiers models et views
Déplacements des fichiers board_mgmtsystem_view.xml, menu.xml et mgmtsystem_system.xml dans views
Renommage de board_mgmtsystem_view.xml en board_mgmtsystem.xml
Déplacements des fichiers mgmtsystem_system.py dans models

Opération effectué dans la vue board_mgmtsystem.xml

Retrait dans board_mgmtsystem_form du code

<hpaned>
    <child1>
    </child1>
    <child2>
    </child2>
</hpaned>

à l'intérieur de la balise form

Opération effectué dans la vue menu.xml

Retrait dans menuitem menu_mgmtsystem_root des paramêtres

web_icon="images/mgmtsystem.png"
web_icon_hover="images/mgmtsystem-hover.png"

modification du fichier __openerp_.py afin qu'il reflète les différents chemin des fichiers

### CODE REVIEW

**Comment by dreispt** [comment]
There a few build warnings to fix: 

```
************* Module mgmtsystem

mgmtsystem/__init__.py:1: [W8201(incoherent-interpreter-exec-perm), ] Incoherent interpreter comment [] and executable permission [True]. Remove interpreter comment or executed `chmod -x myfile.py`

mgmtsystem/__init__.py:1: [C7902(missing-readme), ] Missing ./README.rst file. Template here: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

mgmtsystem/__init__.py:1: [W7902(duplicate-xml-record-id), ] Duplicate xml record id mgmtsystem.system_form

mgmtsystem/__init__.py:1: [W7902(duplicate-xml-record-id), ] Duplicate xml record id mgmtsystem.system_tree

mgmtsystem/__init__.py:1: [W7902(duplicate-xml-record-id), ] Duplicate xml record id mgmtsystem.system_action

mgmtsystem/__init__.py:1: [W7902(duplicate-xml-record-id), ] Duplicate xml record id mgmtsystem.open_board_mgmtsystem

mgmtsystem/__init__.py:1: [W7902(duplicate-xml-record-id), ] Duplicate xml record id mgmtsystem.board_mgmtsystem_form

************* Module mgmtsystem.mgmtsystem_system

mgmtsystem/mgmtsystem_system.py:1: [C8201(no-utf8-coding-comment), ] No UTF-8 coding comment found: Use `# coding: utf-8` or `# -*- coding: utf-8 -*-`

mgmtsystem/mgmtsystem_system.py:29: [C8104(class-camelcase), mgmtsystem_system] Use `CamelCase` "MgmtsystemSystem" in class name "mgmtsystem_system". You can use oca-autopep8 of https://github.com/OCA/maintainer-tools to auto fix it.

************* Module mgmtsystem.__openerp__

mgmtsystem/__openerp__.py:1: [C8201(no-utf8-coding-comment), ] No UTF-8 coding comment found: Use `# coding: utf-8` or `# -*- coding: utf-8 -*-`

mgmtsystem/__openerp__.py:21: [C8103(manifest-deprecated-key), ] Deprecated key "description" in manifest file

************* Module mgmtsystem.tests.test_create_mgmtsystem

mgmtsystem/tests/test_create_mgmtsystem.py:1: [C8201(no-utf8-coding-comment), ] No UTF-8 coding comment found: Use `# coding: utf-8` or `# -*- coding: utf-8 -*-`

************* Module mgmtsystem.models.mgmtsystem_system

mgmtsystem/models/mgmtsystem_system.py:1: [C8201(no-utf8-coding-comment), ] No UTF-8 coding comment found: Use `# coding: utf-8` or `# -*- coding: utf-8 -*-`

mgmtsystem/models/mgmtsystem_system.py:29: [C8104(class-camelcase), mgmtsystem_system] Use `CamelCase` "MgmtsystemSystem" in class name "mgmtsystem_system". You can use oca-autopep8 of https://github.com/OCA/maintainer-tools to auto fix it.
```

File: mgmtsystem/readme.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Migration du module
```
Comment (dreispt) [inline mgmtsystem/readme.md:1]: Please keep and improve the 8.0 README.rst. In english please.

**Comment by dreispt** [comment]
@max3903 The dependency on `document_page` is not really required here. Can we drop it please?

**Comment by dreispt** [comment]
@naousse Thank you for contributing :smile:

**Comment by max3903** [comment]
@dreispt sure

**Comment by dreispt** [comment]
@naousse Please remove the `document_page` dependency

File: mgmtsystem/models/mgmtsystem_system.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    return self.env.user.company_id.id
+
+
+class mgmtsystem_system(models.Model):
+
+    _name = 'mgmtsystem.system'
+    _description = 'System'
+
+    name = fields.Char(string='System', required=True)
+    manual = fields.Many2one('document.page', string='Manual')
```
Comment (max3903) [inline mgmtsystem/models/mgmtsystem_system.py:35]: @dreispt what do you want to do with this field? I suggest to move it to the mgmtsystem_manuals module

Comment (dreispt) [inline mgmtsystem/models/mgmtsystem_system.py:35]: Sorry, I didn't notice that there was this dependency after all.
Your suggestion sounds great.
One nitpick - to fit conventions the module should be named `mgmtsystem_manual`.

**Comment by naousse** [comment]
@dreispt according to your suggestion, i noticed that there is already a mgmtsystem_manuals module.  the mgmtsystem module seems to be the base module for management system. and the view inside this module are also use the manual field that's inside the model.  Do you think that i should move all that code to the mgmtsystem_manuals and leave the mgmtsystem module as a dumb module?

**Comment by max3903** [comment]
@naousse Just move the manual field to move the dependency.

File: mgmtsystem/views/board_mgmtsystem.xml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+            <field name="view_id" ref="board_mgmtsystem_form"/>
+        </record>
+
+        <menuitem
+            id="mgmtsystem_dashboard"
+            name="Management Systems"
+            sequence="50"
+            parent="base.menu_reporting_dashboard"
+            action="open_board_mgmtsystem"/>
+
```
Comment (max3903) [inline mgmtsystem/views/board_mgmtsystem.xml:30]: Remove the 2 views and rename the menu item to Reporting. We will create other menu items in other modules and use the pivot view.

File: mgmtsystem/views/mgmtsystem_system.xml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  <data>
+
+    <record model="ir.ui.view" id="system_form">
+      <field name="name">mgmtsystem_system_form</field>
+      <field name="model">mgmtsystem.system</field>
+      <field name="arch" type="xml">
+        <form string="Management System">
+          <group>
+            <field name="name"/>
+            <field name="manual"/>
```
Comment (max3903) [inline mgmtsystem/views/mgmtsystem_system.xml:12]: Should be added by mgmtsystem_manual module

File: mgmtsystem/views/mgmtsystem_system.xml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+      </field>
+    </record>
+
+    <record model="ir.ui.view" id="system_tree">
+      <field name="name">mgmtsystem_system_tree</field>
+      <field name="model">mgmtsystem.system</field>
+      <field name="arch" type="xml">
+        <tree string="Management Systems">
+          <field name="name"/>
+          <field name="manual"/>
```
Comment (max3903) [inline mgmtsystem/views/mgmtsystem_system.xml:25]: Should be added by mgmtsystem_manual module

**Comment by max3903** [comment]
Superseeded by #94

**Pull request closed without merging**